Gemfile.jruby-1.9.lock.release

```diff
@@ -166,12 +166,12 @@ GEM
       jls-grok (~> 0.11.1)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       logstash-patterns-core
-    logstash-codec-netflow (3.7.1)
+    logstash-codec-netflow (3.8.3)
```

Comment:
Fix a number of stability issues https://github.com/logstash-plugins/logstash-codec-netflow/blob/master/CHANGELOG.md